src/entries/popup/components/CommandK/references.ts

```diff
@@ -6,7 +6,6 @@ export const actionLabels = {
   openInNewTab: () => i18n.t('command_k.action_labels.open_in_new_tab'),
   switchToWallet: () => i18n.t('command_k.action_labels.switch_to_wallet'),
   sendToWallet: () => i18n.t('command_k.action_labels.send_to_wallet'),
-  addContact: () => i18n.t('command_k.action_labels.add_contact'),
```

Comment:
Yeah since we used to do "Add contact" as a shortcut hint instead of "Active command". Added a screenshot below to show how it should be 👇

<img width="339" alt="image" src="https://github.com/rainbow-me/browser-extension/assets/53529533/3c4224ce-8620-4b35-898e-8d77d2b11ec8">